simpleclient_spring_boot/pom.xml

```diff
@@ -42,32 +42,48 @@
             <name>Eliezio Oliveira</name>
             <email>eliezio.oliveira@gmail.com</email>
         </developer>
+        <developer>
+            <id>pidster</id>
+            <name>Stuart Williams</name>
+            <organization>private</organization>
+            <organizationUrl>https://github.com/pidster</organizationUrl>
+        </developer>
     </developers>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
+        <prometheus.client.version>0.0.19-SNAPSHOT</prometheus.client.version>
+        <spring.core.version>4.3.4.RELEASE</spring.core.version>
+        <spring.boot.version>1.4.2.RELEASE</spring.boot.version>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>io.prometheus</groupId>
             <artifactId>simpleclient</artifactId>
-            <version>0.0.19-SNAPSHOT</version>
+            <version>${prometheus.client.version}</version>
```

Comment:
Really?! 
I guess that explains why it's duplicated...
May I ask which ones (for my own future reference)?